cfn-resources/autogen/schema-gen/main.go

```diff
@@ -282,28 +277,15 @@ func readArgs() (compare bool, err error) {
 	return false, nil
 }
 
-func sortProperties(properties map[string]Property) (props map[string]Property) {
-	keys := make([]string, len(properties))
-	for key := range properties {
-		keys = append(keys, key)
-	}
-	sort.Strings(keys)
-	props = make(map[string]Property, len(properties))
-	for _, key := range keys {
-		props[key] = properties[key]
-	}
-	return props
-}
-
-func sortDefinitions(properties map[string]Definitions) (props map[string]Definitions) {
-	keys := make([]string, len(properties))
-	for key := range properties {
-		keys = append(keys, key)
+func sortProperties[V any](properties map[string]V) (props map[string]interface{}) {
```

Comment:
using generics instead

Reply:
[V any] allows me to use this function for both Definitions and Properties. Getting this error when trying to use `map[string]any`. Any suggestions on how to fix it? 
<img width="739" alt="Screenshot 2023-03-20 at 09 01 48" src="https://user-images.githubusercontent.com/5663078/226292710-0e49efcd-97fd-49e9-8a07-642b746e5d4b.png">

Reply:
I will open a follow-up PR to address this comment if needed. Thanks!